packages/Portal/src/Portal.spec.tsx

```diff
@@ -14,12 +14,37 @@ describe('packages/Portal', () => {
     </div>,
   );
 
+  function isElement(el: Node): el is HTMLElement {
+    return el.nodeType === Node.ELEMENT_NODE;
+  }
+
+  function getByIdOrThrow(id: string) {
+    const el = document.getElementById(id);
+
+    if (el == null) {
+      throw new Error(`Could not find element: ${id}`);
+    }
+
+    return el;
+  }
+
   test(`appends portal content to document body`, () => {
-    expect(document.body.firstChild.outerHTML).toBe(
+    const firstChild = document.body.firstChild;
+    const lastChild = document.body.lastChild;
+
+    if (firstChild == null || !isElement(firstChild)) {
```

Comment:
Ditto other PR - why throws and not assertions?

Reply:
Ditto other PR response: TypeScript type narrowing